src/redux/ui/delegated-validators/selectors.ts

```diff
@@ -11,6 +11,6 @@ export const getDelegatedValidators = (
         (state.ui.delegatedValidators &&
             state.ui.delegatedValidators[blockchain] &&
             state.ui.delegatedValidators[blockchain][chainId as string]) ||
-        []
+        undefined
```

Comment:
no need for `|| undefined`. What you are actually doing is : if the value is undefined return undefined.